### DISCUSSION TITLE
Overleaf用Xelatex编译中文变空白

### DISCUSSION THREAD

**Discussion by House-Leo** [sikouhjw/gdutthesis#51] [General]
![image](https://user-images.githubusercontent.com/47940711/159009125-0f6e2e4a-6b72-4535-a640-4a61ef14ae4b.png)
Overleaf用Xelatex编译现在会出现看不到中文只有英文的情况，LuaLatex编译就没问题

**Comment by sikouhjw** [comment]
1. 将 pdf 下载到本地后查看；
2. 如果有中文，有可能是浏览器的问题，尝试更换浏览器、浏览器的 pdf 阅读器；
3. 也可能是 Overleaf 出 bug 了，可以向他们的 support 写信。

**Reply by House-Leo** [reply]
> 1. 将 pdf 下载到本地后查看；
> 2. 如果有中文，有可能是浏览器的问题，尝试更换浏览器、浏览器的 pdf 阅读器；
> 3. 也可能是 Overleaf 出 bug 了，可以向他们的 support 写信。

换成浏览器的pdf viewer就ok了，估计是overleaf pdfviewer的问题，谢谢

**Reply by sikouhjw** [reply]
Overleaf 后续应该是修复了这个问题，不会复现了。
